### ISSUE TITLE
coverage-reports keeps failing on Windows (error mentions clear_expected_if_blessed)

### ISSUE THREAD

**Issue by RalfJung** [rust-lang/rust#105475]
At least 2 PRs are seeing this error
```
 ---- [run-make] src/test\run-make\coverage-reports stdout ----

error: make failed
status: exit code: 2
command: "make"
--- stdout -------------------------------
make[1]: Entering directory '/d/a/rust/rust/src/test/run-make/coverage-reports'
make[1]: Leaving directory '/d/a/rust/rust/src/test/run-make/coverage-reports'
------------------------------------------
--- stderr -------------------------------
Makefile:83: clear_expected_if_blessed: No such file or directory
------------------------------------------
```
Cc https://github.com/rust-lang/rust/pull/104449 https://github.com/rust-lang/rust/pull/105160

**Comment by RalfJung** [comment]
Also see [Zulip discussion](https://rust-lang.zulipchat.com/#narrow/stream/242791-t-infra/topic/run-make.2Fcoverage-reports)

**Comment by ehuss** [comment]
I believe this is now fixed via #105477.
